ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDbFactory.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ */
+
+package com.github.ambry.named;
+
+import com.codahale.metrics.MetricRegistry;
+import com.github.ambry.account.AccountService;
+import com.github.ambry.mysql.MySqlUtils.DbEndpoint;
+import com.github.ambry.config.ClusterMapConfig;
+import com.github.ambry.config.MySqlNamedBlobDbConfig;
+import com.github.ambry.config.VerifiableProperties;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+
+public class MySqlNamedBlobDbFactory implements NamedBlobDbFactory {
+  private final MySqlNamedBlobDbConfig config;
+  private final String localDatacenter;
+  private final MetricRegistry metricRegistry;
+  private final AccountService accountService;
+
+  public MySqlNamedBlobDbFactory(VerifiableProperties verifiableProperties, MetricRegistry metricRegistry,
```

Comment:
May I know which module will call this factory to get the db? Are you planning to call it in IdConverter?

Reply:
Yep, let me add the code to add this into IdConverterFactory.